### DISCUSSION TITLE
update_report_from_reportjson - Missing theme file error

### DISCUSSION THREAD

**Discussion by mlongoria** [microsoft/semantic-link-labs#258] [Q&A] [answered]
I have a report deployed as pbix (not pbir) format. I am a semantic-link-labs newbie, so I might be misunderstanding something. I wanted to replace an old field with a new field. I followed Kurt's tutorial [here](https://data-goblins.com/power-bi/programmatically-modify-reports). But when I run this code, I get an error.
```
rep.update_report_from_reportjson(
    report = w_report.value, 
    report_json = new_report_json, 
    workspace= w_workspace.value
)
```
The error: 
`"error":{"errorCode":"Workload_MissingFileFromDefinition","message":"The following file was not found: 'RegisteredResources/Custom4859761637026365.json'"}}`

The report does have a custom theme that started from a built-in theme and had some colors and fonts changed. I would like to confirm: Is this just a limitation that this function won't work when the report is using a custom theme? Or is this user error? I see that the sempy_labs.report.update_report_from_reportjson function has 3 parameters: report, report_json, and workspace. So I'm not seeing a way to handle the theme. Am I missing somewhere that lists specific limitations like this?

**Comment by m-kovalsky** [comment, marked as answer]
Note that doing this type of operation is completely at your own risk and may corrupt your file. Can look into if the update report definition handles theme files as well for PBIR-Legacy styled report files. Probably best to raise this as an 'enhancement' issue.
